certificate-rotation.md

```diff
@@ -0,0 +1,113 @@
+There are two parts to certificate rotation: generating the new certificates, and reloading them.
```

Comment:
there are generally three reasons to rotate certificates and keys:
* they're expiring soon so you're forced to (should not occur too often with out default of 10 years)
* you have a policy of frequent rotation for security reasons. if you have good PKI infrastructure, this may mean very frequent rotation
* some keys were compromised. rotation is probably not enough here, it might be necessary to just delete all old certs and generate completely new ones. but it could be just one client or node key, in which case rotating might be sufficient.